### DISCUSSION TITLE
I need to create a convenient panel in the left bar for the moderator.

### DISCUSSION THREAD

**Discussion by niknikita** [OrchardCMS/OrchardCore#15982] [Q&A]
I need to create a convenient panel in the left bar for the moderator.

How can I rename section names?
I now have the following picture: https://ibb.co/WsdCvD9

It is very inconvenient to work with this, since the section names are not clear, and I need to rename them.
Dropdown sections are also very inconvenient. I need a fixed display of all items at once.
I just need 4 points in one section with the names:

https://ibb.co/hRJ5KLY

**Comment by Skrypt** [comment]
Well, first you need to create a new user role which will have access to nothing to clean up the left menu for that user role. And then you can create an `admin menu` by enabling that module to customize it. Though, I don't think we can assign an admin menu by role as of now. Maybe there is something missing.

**Comment by niknikita** [comment]
> Well, first you need to create a new user role which will have access to nothing to clean up the left menu for that user role. And then you can create an `admin menu` by enabling that module to customize it. Though, I don't think we can assign an admin menu by role as of now. Maybe there is something missing.

It was partially possible to solve the problem, created an admin menu, but I don’t know how to hide the native menu item “content”

**Comment by Skrypt** [comment]
I'm pretty sure you could also override the INavigationProvider simply in your own module.

**Comment by ns8482e** [comment]
@niknikita Admin Menus are defined using `INavigationProvider` that defines menus for every module and `NavigationManager` builds menus from all registered `INavigationProvider`

If you need advanced customization on Admin Menus, you can create your own `CustomNavigationManager` that implements `INavigationManager` that replace orchard core provided `NavigationManager` in ServiceCollection
